public/index.html

```diff
@@ -2,37 +2,38 @@
+  <!-- For sharing & search engines -->
+  <meta property="og:type" content="website" />
+  <meta property="og:image" content="common/assets/logo-og.png" />
+  <meta property="og:image:width" content="1400" />
+  <meta property="og:image:height" content="1400" />
+  <meta property="og:site_name" content="Mozilla Advocacy">
+  <meta property="og:determiner" content="" />
+  <meta property="og:locale" content="en_US" />
+  <meta property="og:title" content="Mozilla&rsquo;s Policy & Advocacy Program - Home">
+  <meta property="og:url" content="http://advocacy.mozilla.org/">
+  <meta property="description" content="We are building a global movement to protect the free and open Web." />
+  <meta property="og:description" content="We are building a global movement to protect the free and open Web.">
+  <!-- <meta property="og:description" content="We Are Building a Global Movement to Protect the Free and Open Web Mozilla supports the heroes of the Web — the developers, advocates and ..."> -->
 
-    <!-- Home string needs to be fixed or removed -->
-    <meta name="title" content="Home | Mozilla Advocacy">
-    <meta name="description" content="We Are Building a Global Movement to Protect the Free and Open Web Mozilla supports the heroes of the Web — the developers, advocates and ...">
-    <link rel="stylesheet" type="text/css" href="/build/advocacy.css">
-    <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Fira+Sans:300,400,600">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
-    <title>Mozilla Advocacy</title>
-    <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="/assets/favicon/apple-touch-icon-180x180.png" >
-    <link rel="icon" type="image/png" sizes="196x196" href="/assets/favicon/favicon-196x196.png">
-    <link rel="shortcut icon" href="/assets/favicon.ico">
+  <!-- Home string needs to be fixed or removed -->
+  <meta name="title" content="Home | Mozilla Advocacy">
+  <meta name="description" content="We Are Building a Global Movement to Protect the Free and Open Web Mozilla supports the heroes of the Web — the developers, advocates and ...">
+  <link rel="stylesheet" type="text/css" href="/build/advocacy.css">
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Fira+Sans:300,400,600">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+  <title>Mozilla Advocacy</title>
+  <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="/assets/favicon/apple-touch-icon-180x180.png" >
+  <link rel="icon" type="image/png" sizes="196x196" href="/assets/favicon/favicon-196x196.png">
+  <link rel="shortcut icon" href="/assets/favicon.ico">
 </head>
 
 <body>
   <div id="my-app"></div>
   <script src="/build/advocacy.js"></script>
+  <script src="//c.shpg.org/352/sp.js"></script>
```

Comment:
This is a script for a service called share progress. It allows the messages people share to be ab tested. This is just the generic script that gets used later.